irods/manager/user_manager.py

```diff
@@ -188,16 +188,14 @@ def get(self, name, user_zone=""):
         return iRODSUserGroup(self, result)
 
     def create(self, name, user_type='rodsgroup', user_zone="", auth_str=""):
-        message_body = GeneralAdminRequest(
-            "add",
-            "user",
+        message_body = UserAdminRequest(
+            "mkgroup",
             name,
             user_type,
-            "",
-            ""
+            user_zone
         )
         request = iRODSMessage("RODS_API_REQ", msg=message_body,
-                               int_info=api_number['GENERAL_ADMIN_AN'])
+                               int_info=api_number['USER_ADMIN_AN'])
```

Comment:
Did any tests fail with this change?  I'd assume not, but wanted to be sure. It seems reasonable that if a user of type rodsadmin makes a call through USER_ADMIN_AN, the iRODS server goes ahead and considers it a GENERAL_ADMIN_AN. @korydraughn @alanking

Reply:
ah, good point.  might be worth it to confirm that.   or add another test with that scenario.

Reply:
Without this change, the test @MartinFlores751 wrote would fail because of the mismatch between the message body and the API number.

My question is ... Is this change isolated to the groupadmin API of the PRC? Does PRC provide separate implementations for admins and groupadmins? 

To be more clear, does this change affect real admin functionality exposed via the PRC?

Reply:
I think adding a test that is more or less equivalent to the existing test except that a `rodsadmin` is created and performs the actions is a very good idea.

Reply:
@korydraughn No, presently the PRC is agnostic as to user type when an API is performed.